common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -99,6 +99,9 @@ steps:
       NODE_ENV: development
     condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
 
+  - script: npx playwright install chromium
+    displayName:"install chromium for playwright"
```

Comment:
Ya, I realize that. I was thinking maybe we add to Readme that need to install chromium but that is not ideal. Will look into if I can find another approach where the solution is elegant enough where no additional work is required for user and also not hacky like we had with `installPlaywright.js`

Reply:
https://www.youtube.com/watch?v=PxLEjzi9rXQ

Reply:
so how will consumers get this now?

Reply:
@anmolshres98 ^ see comment

Reply:
thx for linking to that table; depending on which pkg we use, we need to manually cleanup. do we need to do that here?

Reply:
did you run certa locally?

Reply:
We are calling browser.close() explicitly yes. I can add the context close too just in case. Yes, I ran rush cover locally and it was working fine(passing)

Reply:
After looking at documentation the browser.close() should suffice for our case, no additional cleanup is needed.